public_html/script.js

```diff
@@ -75,6 +95,8 @@ function processReceiverUpdate(data) {
                                 $(plane.tr).css('font-style', 'italic');
                         } else {
                                 plane.tr.cells[0].textContent = hex;
+                                //append the flag
+				plane.tr.cells[0].appendChild(img);
```

Comment:
Put this in the row template in the static HTML rather than building it dynamically. Then just set the image path after cloning.